src/client.ts

```diff
@@ -298,29 +298,65 @@ interface CurrentPhaseResponse {
 export interface Phase {
   effective?: Date
   features: Features[]
+  trial?: boolean
 }
 
+export interface CancelPhase {}
+
 const isDate = (d: any): d is Date =>
   d && typeof d === 'object' && d instanceof Date
 
 export const isPhase = (p: any): p is Phase =>
   p &&
   typeof p === 'object' &&
   (p.effective === undefined || isDate(p.effective)) &&
+  (p.trial === undefined || typeof p.trial === 'boolean') &&
   Array.isArray(p.features) &&
   !p.features.some((f: any) => !isFeatures(f))
 
-export interface SubscribeRequest {
+export interface CheckoutParams {
+  success_url?: string
+  cancel_url?: string
+}
+
+export interface ScheduleRequest {
   org: OrgName
-  phases?: Phase[]
+  phases?: Phase[] | [CancelPhase]
   info?: OrgInfo
+  checkout?: CheckoutParams
+}
+
+/**
+ * @deprecated alias for ScheduleRequest
+ */
+export type SubscribeRequest = ScheduleRequest
+
+export interface ScheduleResponse {
+  checkout_url?: string
+}
+
+export interface SubscribeOptions {
+  effective?: Date
+  info?: OrgInfo
+  trialDays?: number
+  checkout?: CheckoutParams
```

Comment:
Is the intermixing of XOptions <> XParams intended?

Reply:
Oh yeah, that is weird.  My habit is to call them "BlahOptions" if it's a thing the user passes in, but I think I copied this from the golang types and just left the name untouched.  We can align on "XParams" though, no strong opinions either way.

Reply:
I like Params only because it's fewer letters. Consistency in the library above all else, though.